src/components/Timer.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment'; 
+
+export const Timer = () => {
+  const [duration, setDuration] = useState(0)
+  useEffect(() => {
+    setTimeout(() => {
+      setDuration(duration + 1);
+    }, 1000);
+  }, [duration]);
+  const durationToDisplay = moment.duration(duration, 'seconds').toString()
```

Comment:
Tak naprawdę całość on lini 6 do 12 możesz zawrzeć w jednej funkcji, którą będzie tu tylko używać ;) 

```
const useTimer = () => {
  const [duration, setDuration] = useState(0);
 useEffect(() => {
    setTimeout(() => {
      setDuration(duration + 1);
    }, 1000);
  }, [duration]);

return moment.duration(duration, 'seconds').toString()
}
```

a w tym pliku zrobisz sobie tylko const duration = useTimer();

Reply:
troche zmienilam ten komponent teraz, wiec nie wydzielilam tej czesci kodu do osobnej funkcji. Jesli uwazasz, ze nadal powinnam tak zrobic, to zobacz na nowym PR.